src/main/scala/chisel3/util/Decoupled.scala

```diff
@@ -344,7 +344,6 @@ object Queue {
     *   consumer.io.in <> Queue(producer.io.out, 16)
     * }}}
     */
-  @nowarn("cat=deprecation&msg=TransitName")
```

Comment:
this should have been deleted earlier

Reply:
i deleted one of these off master already, it should have been deleted in the "delete TransitName" PR. But sure I can open a different PR for that.